docs/data/sql_functions.yml

```diff
@@ -1178,6 +1178,22 @@ valueconstruction:
       ```sql
       f(columns => DESCRIPTOR(`col1`, `col2`), on_time => DESCRIPTOR(`ts`))
       ```
+  - sql: OBJECT_OF(className, [key, value [, key, value , ...]])
```

Comment:
keep in mind that Chinese version also should be updated (could be in English for now)